client/src/App.css

```diff
@@ -47,27 +38,33 @@
   margin-right: 30px;
 }
 .canvasContainer {
+  border-radius: 25px;
+  border: 2px solid #21ad4b;
   position: relative;
-  width: 800px;
-  height: 533px;
-  padding-right: 20px;
-  padding-left: 20px;
-  padding-top: 20px;
-  padding-bottom: 20px;
+  width: 65vh;
+  height: 65vh;
+  padding-right: 10px;
+  padding-left: 10px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  margin: auto;
 }
 
 .inputContainer {
-  width: 800px;
-  padding-right: 35px;
-  padding-left: 20px;
+  width: 65vh;
+  padding: 10px;
+  margin: auto;
 }
 
-.timer {
-  position: absolute;
-  top: 10px;
-  right: 35px;
-  z-index: 100;
-  padding: inherit;
+@media only screen and (max-width: 600px) {
```

Comment:
@mahadevans87 : We already have a `div` each for inputContainer and canvasContainer. And those seem to be surrounded by <Grid item>. Is it possible to directly use grid item attributes such as xs, sm etc, instead of media queries ?

Reply:
I am not sure how we can specify different css for different media queries. I saw a material-ui hook where we get info on media queries. But that seemed too much work when we can achieve this explicitly using media queries in css. Is there a better way

Reply:
Okay. I was wondering if it is possible to use the React Grid system itself to specify the width instead of relying on CSS.